integration-cli/docker_api_swarm_test.go

```diff
@@ -371,6 +371,52 @@ func (s *DockerSwarmSuite) TestApiSwarmServicesCreateGlobal(c *check.C) {
 	waitAndAssert(c, defaultReconciliationTimeout, d5.checkActiveContainerCount, checker.Equals, 1)
 }
 
+func (s *DockerSwarmSuite) TestApiSwarmServicesUpdate(c *check.C) {
+	const nodeCount = 3
+	var daemons [nodeCount]*SwarmDaemon
+	daemons[0] = s.AddDaemon(c, true, true)
+	for i := 1; i < nodeCount; i++ {
+		daemons[i] = s.AddDaemon(c, true, false)
+	}
+	time.Sleep(1 * time.Second) // make sure all daemons are ready to accept tasks
```

Comment:
Nit, do for loop on each daemon checking for `d.Status.State == swarm.NodeStateReady`